packages/app/src/widgets/position-section/position-section.tsx

```diff
@@ -10,6 +10,8 @@ import { useLendIncrease } from '@/features/borrow-flow/hooks/use-lend-increase'
 import { formatUsd } from '@/features/borrow-flow/formatters'
 import { PositionCell } from './components/position-cell'
 
+const FULL_POSITION_SIZE = 3
```

Comment:
Is it equal to `SUPPORTED_TOKENS.length`?

Reply:
Thx, edited